variants/lionbits3/pins_arduino.h

```diff
@@ -22,27 +22,27 @@ static const uint8_t BT_LED = 37;
-static const uint8_t D7 = 14;   //I/O   SCl     GPIO14, TOUCH14,ADC2_CH3,   MCPWM,  U0RXD_in,U0CTS_in,U0DSR_in,U0TXD_out,UoRTS_out,U1RXD_in,U1CTS_in,U1DSR_in,U1TXD_out,U1RTS_out,U1DTR_out,U2RXD_in,U2CTS_in,U2DSR_in,U2TXD_out,U2RTS_out,U2DTR_out,LEDPWM
-static const uint8_t D8 = 15;   //I/O           GPIO15,                     MCPWM,  U0RXD_in,U0CTS_in,U0DSR_in,U0TXD_out,UoRTS_out,U1RXD_in,U1CTS_in,U1DSR_in,U1TXD_out,U1RTS_out,U1DTR_out,U2RXD_in,U2CTS_in,U2DSR_in,U2TXD_out,U2RTS_out,U2DTR_out,LEDPWM
-static const uint8_t D9 = 16;   //I/O           GPIO16,                     MCPWM,  U0RXD_in,U0CTS_in,U0DSR_in,U0TXD_out,UoRTS_out,U1RXD_in,U1CTS_in,U1DSR_in,U1TXD_out,U1RTS_out,U1DTR_out,U2RXD_in,U2CTS_in,U2DSR_in,U2TXD_out,U2RTS_out,U2DTR_out,LEDPWM
-static const uint8_t D10 = 17;  //I/O           GPIO17,         ADC2C_H6    MCPWM,  U0RXD_in,U0CTS_in,U0DSR_in,U0TXD_out,UoRTS_out,U1RXD_in,U1CTS_in,U1DSR_in,U1TXD_out,U1RTS_out,U1DTR_out,U2RXD_in,U2CTS_in,U2DSR_in,U2TXD_out,U2RTS_out,U2DTR_out,LEDPWM
-static const uint8_t D11 = 18;  //I/O           GPIO18,         ADC2C_H7    MCPWM,  U0RXD_in,U0CTS_in,U0DSR_in,U0TXD_out,UoRTS_out,U1RXD_in,U1CTS_in,U1DSR_in,U1TXD_out,U1RTS_out,U1DTR_out,U2RXD_in,U2CTS_in,U2DSR_in,U2TXD_out,U2RTS_out,U2DTR_out,LEDPWM
-static const uint8_t D12 = 8;   //I/O           GPIO8,                      MCPWM,  U0RXD_in,U0CTS_in,U0DSR_in,U0TXD_out,UoRTS_out,U1RXD_in,U1CTS_in,U1DSR_in,U1TXD_out,U1RTS_out,U1DTR_out,U2RXD_in,U2CTS_in,U2DSR_in,U2TXD_out,U2RTS_out,U2DTR_out,LEDPWM
-static const uint8_t D13 = 39;  //I/O           GPIO39,                     MCPWM,  U0RXD_in,U0CTS_in,U0DSR_in,U0TXD_out,UoRTS_out,U1RXD_in,U1CTS_in,U1DSR_in,U1TXD_out,U1RTS_out,U1DTR_out,U2RXD_in,U2CTS_in,U2DSR_in,U2TXD_out,U2RTS_out,U2DTR_out,LEDPWM
-static const uint8_t D14 = 40;  //I/O           GPIO40,                     MCPWM,  U0RXD_in,U0CTS_in,U0DSR_in,U0TXD_out,UoRTS_out,U1RXD_in,U1CTS_in,U1DSR_in,U1TXD_out,U1RTS_out,U1DTR_out,U2RXD_in,U2CTS_in,U2DSR_in,U2TXD_out,U2RTS_out,U2DTR_out,LEDPWM
-static const uint8_t D15 = 41;  //I/O           GPIO41,                     MCPWM,  U0RXD_in,U0CTS_in,U0DSR_in,U0TXD_out,UoRTS_out,U1RXD_in,U1CTS_in,U1DSR_in,U1TXD_out,U1RTS_out,U1DTR_out,U2RXD_in,U2CTS_in,U2DSR_in,U2TXD_out,U2RTS_out,U2DTR_out,LEDPWM
-static const uint8_t D16 = 48;  //I/O           GPIO48,                     MCPWM,  U0RXD_in,U0CTS_in,U0DSR_in,U0TXD_out,UoRTS_out,U1RXD_in,U1CTS_in,U1DSR_in,U1TXD_out,U1RTS_out,U1DTR_out,U2RXD_in,U2CTS_in,U2DSR_in,U2TXD_out,U2RTS_out,U2DTR_out,LEDPWM
-static const uint8_t D17 = 21;  //I/O           GPIO21,                     MCPWM,  U0RXD_in,U0CTS_in,U0DSR_in,U0TXD_out,UoRTS_out,U1RXD_in,U1CTS_in,U1DSR_in,U1TXD_out,U1RTS_out,U1DTR_out,U2RXD_in,U2CTS_in,U2DSR_in,U2TXD_out,U2RTS_out,U2DTR_out,LEDPWM
+static const uint8_t D0 = 3;    //RX,GPIO3,MCPWM 
+static const uint8_t D1 = 1;    //TX,GPIO1, MCPWM
+static const uint8_t D2 = 9;    //IO, U1RX,GPIO9,  TOUCH9, ADC1_CH8,MCPWM  
```

Comment:
It would be good to be consistent on the number of spaces around the commas across all of the comments and removing trailing whitespaces.

Reply:
Removed